templates/dockerhub/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM debian AS build
+
+RUN apt-get update && apt-get install -y \
+	libstdc++ \
+	libtool \
+	curl
+
+RUN curl --output clever-tools-latest_linux.tar.gz https://clever-tools.cellar.services.clever-cloud.com/releases/<%= version %>/clever-tools-<%= version %>_linux.tar.gz \
+	&& tar xvzf clever-tools-latest_linux.tar.gz \
```

Comment:
remove latest

Reply:
Now we output the curl result in `clever-tools_linux`